.babelrc

```diff
@@ -1,4 +1,12 @@
 {
-  "presets": [["es2015", { "modules": false }], "react"],
-  "plugins": ["transform-class-properties"]
-}
+	"env": {
+		"production": {
+			"presets": [["es2015", { "modules": false }], "react"],
+			"plugins": ["transform-class-properties"]
```

Comment:
I did try the above before committing but it breaks the build:

`ERROR in ./client/containers/App/index.js
Module build failed: SyntaxError: I:/wwwroot/budgeting-sample-app-webpack2/client/containers/App/index.js: Missing class properties transform.`

I thought it should work as well as the env settings are supposed to just override or merge into other defined settings?

Sorry about the spacing. It had defaulted to 2 tabs.